cefpython/browser.pyx

```diff
@@ -739,9 +739,9 @@ cdef class PyBrowser:
                         bool(pyEvent["focus_on_editable_field"])
             self.GetCefBrowserHost().get().SendKeyEvent(cefEvent)
 
-        cpdef py_void SendMouseClickEvent(self, x, y,
+        cpdef py_void SendMouseClickEvent(self, int x, int y,
```

Comment:
Those flags are exported as they are in cef_types_cef3.pxd. I deleted the comment